tests/fast_tests/test_examples.py

```diff
@@ -247,6 +253,17 @@ def test_multi_highway(self):
         }
         self.run_exp(multiagent_highway, **kwargs)
 
+    # TODO(@evinitsky) enable
```

Comment:
Shouldn't leave this in master if it doesn't work

Reply:
Think it's alright, we have other tests like this that we need to turn on at some point. Not perfect code but this isn't a stable project till a version release anyways